apisix/admin/init.lua

```diff
@@ -15,6 +15,7 @@
 -- limitations under the License.
 --
 local require = require
+local filter = require("apisix.router").filter
```

Comment:
Does module "apisix.router" export the filter function?

Reply:
sorry